freud/environment.pyx

```diff
@@ -565,26 +592,28 @@ cdef class EnvironmentCluster(_MatchEnv):
                 Either a :class:`NeighborList <freud.locality.NeighborList>` of
                 neighbor pairs to use in the calculation, or a dictionary of
                 `query arguments
-                <https://freud.readthedocs.io/en/stable/topics/querying.html>`_
-                (Default value: None).
+                <https://freud.readthedocs.io/en/stable/topics/querying.html>`_.
+                Defines the neighbors used for comparing different particle
+                environments (Default value: None).
             env_neighbors (:class:`freud.locality.NeighborList` or dict, optional):
                 Either a :class:`NeighborList <freud.locality.NeighborList>` of
                 neighbor pairs to use in the calculation, or a dictionary of
                 `query arguments
-                <https://freud.readthedocs.io/en/stable/topics/querying.html>`_
-                This argument is used to define the neighbors of the
-                environment that motifs are registered against. If ``None``,
-                the value provided for ``neighbors`` will be used. (Default
-                value: None).
+                <https://freud.readthedocs.io/en/stable/topics/querying.html>`_.
+                Defines the neighbors used as the environment of each particle.
+                If ``None``, the value provided for ``neighbors`` will be used
+                (Default value: None).
             registration (bool, optional):
                 If True, first use brute force registration to orient one set
                 of environment vectors with respect to the other set such that
                 it minimizes the RMSD between the two sets. Enabling this
                 option incurs a significant performance penalty.
                 (Default value = :code:`False`)
             global_search (bool, optional):
-                 If True, do an exhaustive search wherein the environments of
-                every single pair of particles in the simulation are compared.
+                If True, do an exhaustive search wherein the environments of
+                every single pair of particles are compared.
+                Thus it is equivalent to including all particles in the system
+                in :code:`'neighbors'`.
```

Comment:
If `global_search=True`, are the `neighbors` ignored?

Reply:
I think so. I haven't checked the code though. I believe that's why in #979 @erteich also suggested removing the global search flag. see the last few lines of this [comment](https://github.com/glotzerlab/freud/pull/979#issuecomment-1163219389)

Reply:
Ah yes good call. Can you make another issue for removing `global_search` and also add that to the 3.0 milestone (since that's also a breaking change)?